lib/createPartialStylelintResult.mjs

```diff
@@ -22,9 +23,12 @@ export default function createPartialStylelintResult(postcssResult, cssSyntaxErr
 			}
 		}
 
-		const deprecationMessages = postcssResult.messages.filter(
-			(message) => message.stylelintType === 'deprecation',
-		);
+		// @ts-ignore -- TS2339: Property 'noDeprecation' does not exist on type 'Process'.
```

Comment:
[suggest] I think `@ts-expect-error` has more advantages. See:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#ts-ignore-or-ts-expect-error

```suggestion
		// @ts-expect-error -- TS2339: Property 'noDeprecation' does not exist on type 'Process'.
```

Reply:
We don't need to release a patch version. Once a `@types/node` bump PR comes and `@ts-expect-error` becomes an error, we just need to remove the `@ts-expect-error` comment in the bump PR.

If using `@ts-ignore`, it's hard for us to notice the needless `@ts-expect-error`.

Reply:
> We don't need to release a patch version. 
> […] we just need to remove the `@ts-expect-error` comment in the bump PR.

That would be a patch release :)
It seems you are confident about it, let's change it then.

Reply:
> That would be a patch release :)

I don't know why just removing the `@ts-expect-error` comment will be a patch release. 🤷🏼